src/components/dashboardCanvas/Canvas.jsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import "./dashboardCanvas.css";
+var log = console.log;
+var error = console.error
+// FSBL.addEventListener("onReady", function () {
+
+export default class Canvas extends React.Component {
```

Comment:
I see that the canvas is resizable via drag, but it can be dragged outside of the window bounds. At the very least, if there's no fluid way to handle resizing of the window while that drag handler is being dragged, we should reset the bounds of the canvas on mouseup so that the window will contain all of the canvas.